src/oscar/apps/order/abstract_models.py

```diff
@@ -698,10 +698,10 @@ def description(self):
             value = attribute.value
             if isinstance(value, list):
                 ops.append(
-                    "%s = '%s'" % (attribute.type, (", ".join([str(v) for v in value])))
+                    "%s = '%s'" % (attribute.option.name, (", ".join([str(v) for v in value])))
```

Comment:
I think this should be attribute.value because the option could be empty

Reply:
When you look at another abstract_models. For example at baskets module, it is als like i changed.
https://github.com/django-oscar/django-oscar/blob/c20ef16274642dbc95d7ef26d9c6316ce26a08dc/src/oscar/apps/order/abstract_models.py#L690